### ISSUE TITLE
Remove empty style attribute on line 128 in vr/index.html

### ISSUE THREAD

**Issue by akanbifatimah** [publiclab/infragram#257]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted)[](https://github.com/publiclab/infragram/blob/main/.github/first-timers-issue-template.md#-what-you-will-need-to-know)[](https://github.com/publiclab/infragram/blob/main/.github/first-timers-issue-template.md#-step-by-step) issues. Thanks!

🤔 What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

📋 Step by Step
 🙋 Claim this issue: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
💡 Learn how to claim 🙋
 📝 Update the file [index.html](https://github.com/publiclab/infragram/blob/main/vr/index.html#L128) in the [infragram](https://github.com/publiclab/infragram) repository (press the little pen Icon) and edit the line as shown below.
[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a -) which lines to remove, and in green (and a +) which lines to add:

```diff
@@ -125,7 +125,7 @@
            
-          <div class="zone last" id="save-zone" style="">
+          <div class="zone last" id="save-zone" >        


```
 💾 Commit your changes

 🔀 Start a Pull Request. There are two ways how you can start a pull request:

If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.

If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)[](https://github.com/publiclab/infragram/blob/main/.github/first-timers-issue-template.md#please-keep-us-updated)[](https://github.com/publiclab/infragram/blob/main/.github/first-timers-issue-template.md#is-someone-else-already-working-on-this) and open a pull request from there.

 🏁 Done Ask in comments for a review :)
Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update within a week of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

Is someone else already working on this?
🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- If someone seems stuck, offer them some help! Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all)[](https://github.com/publiclab/infragram/blob/main/.github/first-timers-issue-template.md#-questions). Thanks!

🤔❓ Questions?
Leave a comment below!

**Comment by TildaDares** [comment]
Hi @akanbifatimah, this looks really great but I thought you already resolved this in #240

**Comment by akanbifatimah** [comment]
Its the same issue but in different folders @TildaDares

**Comment by akanbifatimah** [comment]
#240 is in pi folder but @257 is in vr folder

**Comment by TildaDares** [comment]
Reserved for @KemiFrank

**Comment by govindgoel** [comment]
@TildaDares #260 is already reserved for @Kemifrank.

**Comment by TildaDares** [comment]
Thanks @govindgoel I’ll assign it to someone else

**Comment by TildaDares** [comment]
Reserved for @Arup-Dep for 24 hours.

**Comment by Arup-Deb** [comment]
Thanks, @TildaDares  For assigning me.

**Comment by TildaDares** [comment]
Resolved in #273.
